requirements.txt

```diff
@@ -3,6 +3,7 @@
 # For testing
 pytest==7.2.1
 pytest-dotenv==0.5.2
+hypothesis==6.67.1
```

Comment:
why do we need this?

Reply:
removed.

i thought it was a requirement for running the tests.